thrift/thrift-gen/template.go

```diff
@@ -43,6 +43,7 @@ func parseTemplate(contents string) (*template.Template, error) {
 		"contextType":   contextType,
 		"goPrivateName": goName,
 		"goPublicName":  goPublicName,
+		"goType":        func() string { return "" },
```

Comment:
let's make a separate function `dummyGoType` and use that. The `dummyGoType` function should have a comment explaining why it's needed and that it will be replaced.